components/warn_metric_ack_modal/warn_metric_ack_modal.tsx

```diff
@@ -141,51 +176,153 @@ export default class WarnMetricAckModal extends React.PureComponent<Props, State
             />
         );
-        const descriptionText = (
-            <FormattedMessage
-                id='warn_metric_ack_modal.number_of_active_users.description'
-                defaultMessage='Mattermost strongly recommends that deployments of over {limit} users upgrade to Mattermost Enterprise Edition, which offers features such as user management, server clustering, and performance monitoring'
-                values={{
-                    limit: this.props.warnMetricStatus.limit,
-                }}
-            />
-        );
 
+        let descriptionText;
+        let subText;
         const learnMoreLink = 'https://mattermost.com/pl/default-admin-advisory';
-        const subText = (
-            <div
-                style={{display: 'flex', opacity: '0.56', flexWrap: 'wrap'}}
-                className='help__format-text'
-            >
-                <FormattedMessage
-                    id='warn_metric_ack_modal.number_of_active_users.subtext'
-                    defaultMessage='By clicking Acknowledge, you will be sharing your information with Mattermost Inc., to learn more about upgrading. {link}'
-                    values={{
-                        link: (
-                            <ErrorLink
-                                url={learnMoreLink}
-                                messageId={t('warn_metric_ack_modal.learn_more.link')}
-                                defaultMessage={'Learn More'}
-                            />
-                        ),
-                    }}
-                />
-            </div>
-        );
 
-        const buttonText = (
-            <FormattedMessage
-                id='warn_metric_ack_modal.contact_support'
-                defaultMessage='Acknowledge'
-            />
-        );
+        if (!isE0Edition) {
+            if (this.props.warnMetricStatus.id === WarnMetricTypes.SYSTEM_WARN_METRIC_NUMBER_OF_ACTIVE_USERS_500) {
+                descriptionText = (
+                    <FormattedMessage
+                        id='warn_metric_ack_modal.number_of_active_users.description'
+                        defaultMessage='Mattermost strongly recommends that deployments of over {limit} users upgrade to Mattermost Enterprise Edition, which offers features such as user management, server clustering, and performance monitoring'
+                        values={{
+                            limit: this.props.warnMetricStatus.limit,
+                        }}
+                    />
+                );
+                subText = (
+                    <div
+                        style={containerStyles}
+                        className='help__format-text'
+                    >
+                        <FormattedMessage
+                            id='warn_metric_ack_modal.number_of_active_users.subtext'
+                            defaultMessage='By clicking Acknowledge, you will be sharing your information with Mattermost Inc., to learn more about upgrading. {link}'
```

Comment:
I think "to learn more about upgrading" shouldn't be part of the previous sentence and then link shouldn't be after the period.

Reply:
I'll defer to @esadur and @jasonblais on this, they review the texts.

Reply:
@mkraft are you suggesting that "to learn more about upgrading" should be removed entirely? And that the link should appear before the period?

Reply:
Removed the phrase "to learn more about upgrading" in the text copy doc.